bin/cmd.js

```diff
@@ -833,13 +829,15 @@ function drawTorrent (torrent) {
   }
 }
 
-function torrentDone () {
+function torrentDone (torrent) {
```

Comment:
If we need a reference to `torrent` then I think this function should be moved so that it captures it.